src/main.py

```diff
@@ -102,17 +109,30 @@ def import_pass(self, pass_file):
             self.window().show_toast(str(exception))
 
     def on_about_action(self, widget, __):
-        about = Adw.AboutWindow()
-        about.set_application_icon('me.sanchezrodriguez.passes')
-        about.set_application_name(_('Passes'))
-        about.set_copyright('Copyright © 2022-2023 Pablo Sánchez Rodríguez')
-        about.set_license_type(Gtk.License.GPL_3_0)
-        about.set_developer_name('Pablo Sánchez Rodríguez')
-        about.set_issue_url('https://github.com/pablo-s/passes/issues')
-        about.set_version('0.9')
-        about.set_website('https://github.com/pablo-s/passes')
-        about.set_transient_for(self.window())
+        about = Adw.AboutWindow(
+            application_icon = Config.APPID,
+            application_name = _('Passes'),
+            copyright = 'Copyright © 2022-2023 Pablo Sánchez Rodríguez',
+            license_type = Gtk.License.GPL_3_0,
+            developer_name = 'Pablo Sánchez Rodríguez',
+            issue_url = 'https://github.com/pablo-s/passes/issues',
+            version = Config.VERSION,
+            developers = [ 'Pablo Sánchez Rodríguez' ],
+            # Translators: do one of the following, one per line: Your Name, Your Name <email@email.org>, Your Name https://websi.te
+            translator_credits = _("translator-credits"),
+            website = 'https://github.com/pablo-s/passes',
+            debug_info=self.troubleshooting,
+            transient_for = self.window()
+        )
+
+        about.add_credit_section(_("Contributors"), [
+            # Contributors: do one of the following, one per line: Your Name, Your Name <email@email.org>, Your Name https://websi.te
+            "Hari Rana",
```

Comment:
Thanks for adding me as a contributor :)
```suggestion
            "Hari Rana (TheEvilSkeleton) https://tesk.page",
```